lint

```diff
@@ -113,7 +113,7 @@ lint_sh() {
     local filename="$1"
     local lint_result=0
 
-    if ! diff <(shfmt -i 4 "${filename}") "${filename}" >/dev/null; then
+    if ! diff <(shfmt -i 4 "${filename}") "${filename}"; then
```

Comment:
This seems equivalent (diff doesn't seem to be used). However, I wonder why it was introduced in the first place.